### DISCUSSION TITLE
Trying to clean reinstall on Big Sur

### DISCUSSION THREAD

**Discussion by kelleydac** [Homebrew/discussions#238] [Getting started]
I had homebrew working on Catalina and recently upgraded to Big Sur. I tried to remove and reinstall home-brew from scratch using the installer script, as shown below, but cannot get past the "Bad file descriptor" error:
 ~ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
==> This script will install:
/usr/local/bin/brew
/usr/local/share/doc/homebrew
/usr/local/share/man/man1/brew.1
/usr/local/share/zsh/site-functions/_brew
/usr/local/etc/bash_completion.d/brew
/usr/local/Homebrew

Press RETURN to continue or any other key to abort
==> /usr/bin/sudo /bin/mkdir -p /Users/dack/Library/Caches/Homebrew
==> /usr/bin/sudo /bin/chmod g+rwx /Users/dack/Library/Caches/Homebrew
==> /usr/bin/sudo /usr/sbin/chown dack /Users/dack/Library/Caches/Homebrew
==> Downloading and installing Homebrew...
fatal: unable to connect to github.com:
github.com[0: 192.30.255.112]: errno=Bad file descriptor

Failed during: git fetch origin --force

**Comment by carlocab** [comment]
I think there's a problem with your connection. See https://github.com/Homebrew/discussions/discussions/190#discussioncomment-143475
